src/Site.js

```diff
@@ -385,6 +412,193 @@ Site.prototype.createPage = function (config) {
   });
 };
 
+/**
+ * Converts an existing Github wiki or docs folder to a MarkBind website.
+ */
+Site.prototype.convert = function (rootPath) {
+  return new Promise((resolve, reject) => {
+    this.addIndexPage(rootPath)
+      .then(() => this.addAboutPage(rootPath))
+      .then(() => this.addFooterToDefaultLayout(rootPath))
+      .then(() => this.addSiteNavToDefaultLayout(rootPath))
+      .then(() => this.addDefaultLayoutToSiteConfig(rootPath))
+      .then(() => this.addTopNavToAddressablePages(rootPath))
+      .then(resolve)
+      .catch(reject);
+  });
+};
+
+/**
+ * Adds an index page to site if not present.
+ * @param rootPath
+ */
+Site.prototype.addIndexPage = function (rootPath) {
+  const indexPagePath = path.join(rootPath, INDEX_MARKDOWN_FILE);
+  const fileNames = ['README.md', 'Home.md'];
+  for (let i = 0; i < fileNames.length; i += 1) {
+    const filePath = path.join(rootPath, fileNames[i]);
+    if (fs.existsSync(filePath)) {
+      fs.copyAsync(filePath, indexPagePath)
+        .catch(() => Promise.reject(new Error(`Failed to copy over ${fileNames[i]}`)));
```

Comment:
This rejected promise never gets passed down the promise chain in `Site.convert`

Reply:
Also, I'm pretty sure `i` will always be `fileNames.length` in this arrow function.